### PR TITLE
added instructions for reviewer - animals part 2

### LINKED ISSUE #78: Create "##Instructions for reviewer" for project: Animals Part 2. Adding Tests
Please add a section to the bottom of this project's _index.md file.
The section heading should be EXACTLY "## Instructions for reviewer"

Basically we have to set reviewers up for success by telling them what success looks like and what to pay attention to when reviewing this project submission.

Description:

- [ ] Look at a bunch of old reviews for this project for each cohort

- [ ] Find the common feedback [This should help you get an idea of what instructions to create]

- [ ] Write them out clearly

- [ ] The section heading should be EXACTLY "## Instructions for reviewer"

### PR DESCRIPTION
**Pull request by Andy-Nkumane**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Vuyisanani — APPROVED** [review]

File: content/projects/oop/animals/part2/_index.md [hunk 1]
```diff
@@ -161,3 +161,11 @@ Test -> Does cat eat food should Fail
 This section is not compulsory. If you do this we'll think you're cool.
 
 Add some extra tests for your Home class. Make sure that it makes all the right noises. And that you can't adopt the same pet twice.
+
+## Instructions for Reviewers
+
+- There should be no global variable.
```
Comment (TheAbstract) [inline content/projects/oop/animals/part2/_index.md]:
```suggestion
- There should be no global variables.
```

File: content/projects/oop/animals/part2/_index.md [hunk 2]
```diff
@@ -161,3 +161,11 @@ Test -> Does cat eat food should Fail
...
+- `self` should not be overused, if a variable is only used in one place then it is not necessary to use `self`.
```
Comment (TheAbstract) [inline content/projects/oop/animals/part2/_index.md]: I don't think the issue here is the number of times the variable is used but how. If a variable is only used within a given method then constantly referring to it as `self.variable` isn't necessary.

**Review by TheAbstract — CHANGES_REQUESTED** [review]
Remember that this project is shared across languages so the instructions need to be general enough to make sense for everyone. Maybe ask one js/java person to see if these make sense and don't cause any confusion.

File: content/projects/oop/animals/part2/_index.md [hunk 3]
```diff
@@ -161,3 +161,11 @@ Test -> Does cat eat food should Fail
 This section is not compulsory. If you do this we'll think you're cool.
 
 Add some extra tests for your Home class. Make sure that it makes all the right noises. And that you can't adopt the same pet twice.
+
+## Instructions for Reviewers
+
+- There should be no global variables.
+- If a variable is only used within a given method, then constantly referring to it as `self.variable`/`this.variable` isn't necessary.
+- `eats()` method should return "Food" and not print.
+- `Dog` and `Cat` class `sounds()` should return their respective sounds "Bark" and "Meow".
```
Comment (Sbonelo01) [inline content/projects/oop/animals/part2/_index.md:170]: This feels like a repetition of what's already in the instructions

**Review by Sbonelo01 — COMMENTED** [review]
A point you could try to emphasize is on this information:

`//Dog Tests`
`Test -> Does dog eat Food should Pass`
`Test -> Does dog eat food should Fail`
`//Cat Tests`
`Cat cat = new Cat();`
`Test -> Does cat Bark should Fail`
`Test -> Does cat Meow should Pass`
`Test -> Does cat eat meat should Fail`
`Test -> Does cat eat Food should Pass`
`Test -> Does cat eat food should Fail`

File: content/projects/oop/animals/part2/_index.md (hunk 3 above)
Comment (TheAbstract) [reply content/projects/oop/animals/part2/_index.md:170]: True but I've seen people mess this up, they either use the Animals `sounds` method or do some other odd thing. Doest hurt emphasising it.

**Review by TheAbstract — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**